javascript/ql/test/library-tests/TypeScript/RegressionTests/ImportSelf/test.ql

```diff
@@ -0,0 +1,3 @@
+import javascript
+
+from Expr e select e, e.getType()
```

Comment:
Would it be worth adding a comment to this test case explaining what it is actually testing?